scripts/daam_script.py

```diff
@@ -143,7 +143,7 @@ def run(self,
                 num_input = len(p.sd_model.model.diffusion_model.input_blocks)
                 num_output = len(p.sd_model.model.diffusion_model.output_blocks)
                 self.tracers = [trace(p.sd_model, p.height, p.width, context_size, layer_idx=i) for i in range(num_input + num_output + 1)]
-                self.attn_captions = [f"IN{i:02d}" for i in range(num_input)] + ["MID"] + [f"OUT{i:02d}" for i in range(num_output)]
```

Comment:
also, this code may generate not appropriate captions if middle layer doesn't have attention block